bokehjs/src/lib/models/glyphs/glyph.ts

```diff
@@ -142,17 +142,17 @@ export abstract class GlyphView extends View {
     const n = pts.length
     if (pts_location == 'center') {
       const halfspan = map(spans, (d) => d/2)
-      pt0 = new NumberArray(n)
+      pt0 = new Array(n)
```

Comment:
For the sake of consistency I would use typed arrays exclusively, in this case `Float64Array`, though it won't have any measurable run time effect.

Reply:
Sure that seems reasonable